go.mod

```diff
@@ -2,6 +2,9 @@ module github.com/crossplane/crossplane
 
 go 1.19
 
+// TODO(negz): Cut a new c-r version, bump before we merge this.
+replace github.com/crossplane/crossplane-runtime => github.com/negz/crossplane-runtime v0.0.0-20230124230124-2ac6ad82f7df
```

Comment:
Adding a note to make sure we don't merge this before https://github.com/crossplane/crossplane-runtime/pull/375.